.pre-commit-config.yaml

```
@@ -218,39 +218,26 @@ repos:
     - tests/
     pass_filenames: false
 
-- repo: local
+- repo: https://github.com/PyCQA/pylint
```

Consistency with the URL style:
```suggestion
- repo: https://github.com/PyCQA/pylint.git
```